### PR TITLE
Fix(18n fr)

### PR DESCRIPTION
**Pull request by AugustinMauroy**
"update" of the French contents

### CODE REVIEW

**Comment by ghost** [comment]
Kind suggestion: You don't need to put English version files here but just to keep French ones ;).

**Comment by ovflowd** [comment]
The huge amount of file changes makes it confusing to know what was changed... As @MaledongGit mentioned, could you only include the actual French changes here? 👀 

Also, is this a fix? What exactly are you fixing here? Could you add a description of what exactly you're doing here? Ty! 🙇 

Also cc @nodejs/nodejs-fr

**Comment by AugustinMauroy** [comment]
@MaledongGit I forgot to remove them. I copied and pasted from the blog folder

**Comment by ghost** [comment]
> @MaledongGit I forgot to remove them. I copied and pasted from the blog folder

Thus you can make a reset in either soft mode to remove those unchanged files out of Git, and then rebase to make a new submit forcely :)

Maybe those commands will help you:

1) https://git-scm.com/docs/git-reset
2) https://git-scm.com/docs/git-rebase

**Comment by ovflowd** [comment]
Hey @AugustinMauroy the CI is still failing, you might need to re-lint the files.

**Comment by AugustinMauroy** [comment]
@ovflowd i di it but I don't understand error
```bash
locale/fr/about/index.md
   40:1-40:71  warning  Found unused definition  no-unused-definitions  remark-lint
   43:1-43:62  warning  Found unused definition  no-unused-definitions  remark-lint
   44:1-44:62  warning  Found unused definition  no-unused-definitions  remark-lint
   45:1-45:43  warning  Found unused definition  no-unused-definitions  remark-lint
   ```
   can you explain ?

**Comment by ovflowd** [comment]
> can you explain ?

You have footnote links that are not being used. You probably translated the links and forgot to do the same on their reference.

Check out the "File changes" tab.

**Comment by AugustinMauroy** [comment]
I looked at it but I don't understand why there is an error

File: locale/fr/about/governance.md [hunk 1]
```diff
@@ -11,14 +11,28 @@ Le projet est co-dirigé par un Comité de Pilotage Technique (Technical Steerin
...
+Le projet est dirigé conjointement par le [Comité directeur technique (CDT)][]
+qui est responsable de l'orientation de haut niveau du projet, et le
+[Comité de la communauté (CommComm)][] qui est responsable de l'orientation et de l'extension de la communauté Node.js.
+l'extension de la communauté Node.js.
+
+[collaborator-guide.md]: https://github.com/nodejs/node/blob/main/doc/contributing/collaborator-guide.md
+[Community Committee (CommComm)]: https://github.com/nodejs/community-committee/blob/master/Community-Committee-Charter.md
+[Consensus Seeking]: https://en.wikipedia.org/wiki/Consensus-seeking_decision-making
+[README.md]: https://github.com/nodejs/node/blob/main/README.md#current-project-team-members
+[Technical Steering Committee (TSC)]: https://github.com/nodejs/TSC/blob/master/TSC-Charter.md
```
Comment (nschonni) [inline locale/fr/about/governance.md:36]:
```suggestion
[Comité directeur technique (CDT]: https://github.com/nodejs/TSC/blob/master/TSC-Charter.md
```

File: locale/fr/about/governance.md [hunk 2]
```diff
@@ -11,14 +11,28 @@ Le projet est co-dirigé par un Comité de Pilotage Technique (Technical Steerin
...
+## Comités de haut niveau
 
-Les personnes proposant des contributions significatives sont faites Collaborateurs et se voient accorder les droits d'écriture sur le projet. Ces personnes sont identifiées par le TSC et leur ajout aux Collaborateurs est discuté lors des réunions hebdomadaires du TSC
+Le projet est dirigé conjointement par le [Comité directeur technique (CDT)][]
+qui est responsable de l'orientation de haut niveau du projet, et le
+[Comité de la communauté (CommComm)][] qui est responsable de l'orientation et de l'extension de la communauté Node.js.
+l'extension de la communauté Node.js.
+
+[collaborator-guide.md]: https://github.com/nodejs/node/blob/main/doc/contributing/collaborator-guide.md
+[Community Committee (CommComm)]: https://github.com/nodejs/community-committee/blob/master/Community-Committee-Charter.md
```
Comment (nschonni) [inline locale/fr/about/governance.md:33]:
```suggestion
[Comité de la communauté (CommComm)]: https://github.com/nodejs/community-committee/blob/master/Community-Committee-Charter.md
```

**Review by nschonni — COMMENTED** [review]
Didn't go through them all, but anchor link text must mach the `[WORDS][]` format, or use `[TRANSLATED_TXT][ORIGINAL_ENGLISH]` format

**Comment by ghost** [comment]
@nodejs/nodejs-fr : Anyone helps to review？

**Comment by AugustinMauroy** [comment]
No need to know that the content is going to change.

**Pull request closed without merging**